src/envoy/token/token_info.h

```diff
@@ -0,0 +1,54 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include "common/common/logger.h"
+#include "envoy/common/pure.h"
+#include "envoy/http/message.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Token {
+
+// For APIs that have no expiration time, use this.
+constexpr std::chrono::seconds kDefaultTokenExpiry(3599);
+
+typedef std::function<void(absl::string_view)> UpdateTokenCallback;
+typedef std::function<std::string()> GetTokenFunc;
```

Comment:
These two functions should not be here.  They are not used by this interface.

Reply:
Done